src/components/atoms/PhoneNumberInput/PhoneNumberInput.tsx

```diff
@@ -39,6 +41,13 @@ const PhoneNumberInput = forwardRef<HTMLInputElement, FormattedInputProps>(
     const textValueRef = useRef<NumberLike>();
     const [flag, setFlag] = useState(getFlagEmoji("IN"));
 
+    useEffectOnce(() => {
+      void import("country-flag-emoji-polyfill").then((module) => {
```

Comment:
whats the benefit of dynamic import here?

Reply:
No particular benefit. Let me change this.